src/video_core/renderer_opengl/gl_rasterizer.cpp

```diff
@@ -678,8 +689,52 @@ void RasterizerOpenGL::InvalidateRegion(VAddr addr, u64 size) {
     query_cache.InvalidateRegion(addr, size);
 }
 
+void RasterizerOpenGL::OnCPUWrite(VAddr addr, u64 size) {
+    MICROPROFILE_SCOPE(OpenGL_CacheManagement);
+    if (!addr || !size) {
```

Comment:
```suggestion
    if (addr == 0 || size == 0) {
```

Since these are integer types.